config/settings.yml

```diff
@@ -1204,6 +1204,7 @@
       :starting_timeout: 10.minutes
       :stopping_timeout: 10.minutes
     :embedded_ansible_worker:
+      :starting_timeout: 20.minutes
```

Comment:
In my tests on a VM running in workstation with 6GB of ram and no other workers running the initial configuration took about 7 minutes and the subsequent starts took about 1 minute.

I'm okay with this value, but we could probably make it smaller if we have a good reason to.
Thoughts?